scripts/parse.sh

```diff
@@ -28,7 +28,7 @@ python3 scripts/combine.py
 python3 scripts/reformat.py
 
 # Copy canteens.json in the output directory:
-echo "Coppying canteens..."
+echo "Copying canteens..."
 cp src/canteens.json $OUT_DIR
```

Comment:
Should also be generated through the new enums.
Additionally, we should delete the `canteens.json`, to not have the information stored double, and to prevent errors, when changes are made to these.

Reply:
True. But don't we need the old file for backwards compability?

Reply:
I'd say the format of the file is alright, so ideally I would propose, that you use the same format, and maybe (for now) also copy your newly generated file to the `canteens.json` file?

Anyway, we could also let the old file there for now, however, I think we should not copy it anymore, as there shouldn't be a canteens.json file in the master branch anymore.

Reply:
Well, I know copy it from ENUM_DIR/canteens.json